pycbc/detector.py

```diff
@@ -62,14 +63,45 @@ def get_available_detectors():
 class Detector(object):
     """A gravitational wave detector
     """
-    def __init__(self, detector_name):
+    def __init__(self, detector_name, reference_time=1126259462):
+        """ Create class representing a gravitational-wave detector
+
+        Parameters
+        ----------
+        detector_name: str
+            The two character detector string, i.e. H1, L1, V1, K1, I1
+        reference_time: str
+            Default is . In this case, the earth's rotation
```

Comment:
Default is what?

Reply:
It's set to the time of GW150914. Comment updated.